Add `decode_prefix_id` class method

TastyPi opened the pull request:
It's useful to be able to convert a prefix ID to a standard ID. For example, I want to be able to do something like this in a single SQL query:

```ruby
class Foo < ActiveRecord::Base
  has_prefix_id :foo
  has_one :bar
end

class Bar < ActiveRecord::Base
  has_prefix_id :bar
  belongs_to :foo
end

foo_prefix_ids = # Get list of prefix IDs from external system

# List the Bars that do not belong to any of the Foos
bars = Bar.where.not(foo_id: foo_prefix_ids.map { |id| Foo.unprefix_id(id) })
```

excid3 commented:
This is a great idea @TastyPi 👍 

I'm wondering if a name like `decode_prefix_id` would be better?

excid3 commented:
Agreed! My thought was "unprefix" sounds like it removes the `user_` from `user_abce`, so `decode` felt like a better description of what it's doing there.

Is there any value in also adding `decode_prefix_ids(prefix_ids_array)` as well? It's easy enough to map, but if that's a common thing you're doing, then it may not hurt to have that method too?

excid3 commented:
Also would you mind adding a note to the CHANGELOG?

TastyPi commented:
Added `decode_prefix_ids` and updated the CHANGELOG. I wouldn't say I do this often, it's the first time I've needed it after 2 years of using this library, but it seems reasonable to include.

excid3 commented:
Thank you! This is great. 👍

excid3 commented:
Just released a new version with this in it.
